tests/test_defects.py

```diff
@@ -0,0 +1,151 @@
+        calibDir = os.path.join(getPackageDir('ci_cpp_gen2'), "DATA", "calibs")
+        butler = dafPersist.Butler(repoDir, calibRoot=calibDir)
+
+        self.config = ipIsr.IsrTaskConfig()
+        self.config.doSaturation = True
+        self.config.doSuspect = True
+        self.config.doSetBadRegions = True
+        self.config.doOverscan = True
+        self.config.doBias = True
+        self.config.doVariance = True
+        self.config.doDark = True
+        self.config.doFlat = True
+        self.config.doDefect = True
+
+        self.config.doLinearize = False
+        self.config.doCrosstalk = False
+        self.config.doWidenSaturationTrails = False
+        self.config.doBrighterFatter = False
+        self.config.doSaturationInterpolation = False
+        self.config.doStrayLight = False
+        self.config.doApplyGains = False
+        self.config.doFringe = False
+        self.config.doMeasureBackground = False
+        self.config.doVignette = False
+        self.config.doAttachTransmissionCurve = False
+        self.config.doUseOpticsTransmission = False
+        self.config.doUseFilterTransmission = False
+        self.config.doUseSensorTransmission = False
+        self.config.doUseAtmosphereTransmission = False
+
+        self.isrTask = ipIsr.IsrTask(config=self.config)
+        # This is not an independent frame.
+        self.dataRef = butler.dataRef('raw', dataId={'detector': 0, 'expId': 2020012800028})
+        results = self.isrTask.runDataRef(self.dataRef)
+        self.exposure = results.outputExposure
+
+    def test_masterFrameLevel(self):
+        """Test image Mean
+
+        Notes
+        -----
+        DMTN-101 3.2
+
+        For each master frame, confirm that the median level is 1.0 to
+        within statistical noise (after masking known defects)
+
+        """
+        mean = afwMath.makeStatistics(self.exposure.getImage(), afwMath.MEAN).getValue()
+        median = afwMath.makeStatistics(self.exposure.getImage(), afwMath.MEDIAN).getValue()
+        sigma = afwMath.makeStatistics(self.exposure.getImage(), afwMath.STDEV).getValue()
+        print("3.2", mean, sigma)
+        self.assertLess(np.abs(mean/median - 1.0), sigma)
+
+    def test_masterFrameSigma(self):
+        """Ill defined.
+
+        Notes
+        -----
+        DMTN-101 3.3
+
+        For each master frame, divide by the standard deviation image,
+        and confirm that the 5-sigma clipped standard is 1.0 to within
+        statistical noise (after masking known defects)
+        """
+        pass
+
+    def test_pixelConsistency(self):
+        """Ill-defined.
+
+        Notes
+        -----
+        DMTN-101 3.4
+
+        for each pixel, check if all master frames are consistent to
+        within statistical noise; if not, confirm that they are
+        included in the defect map.  Also identify pixels in the
+        defect map that are not identified.
+
+        """
+        pass
```

Comment:
Why are they ill-defined and, in that case, are there plans to propose well-defined tests and update DMTN-101?

Reply:
I've deleted the ill defined tests, and will assume a future ticket will add in tests that are defined better.